docs/installation-quickstart.rst

```diff
@@ -0,0 +1,219 @@
+Installation Quickstart
+=======================
+
+This process will install Girder's prerequisites for a Python 2 environment on common systems.
+
+Basic System Prerequisites
+--------------------------
+
+.. tabs::
+   .. group-tab:: Ubuntu 16.04
+
+      To install basic system prerequisites, run the command:
+
+      .. code-block:: bash
+
+         sudo apt-get install -y python-pip python-virtualenv git
+
+      To install system prerequisites for Girder's ``ldap`` plugin, run the command:
+
+      .. code-block:: bash
+
+         sudo apt-get install -y libldap2-dev libsasl2-dev
+
+   .. group-tab:: Ubuntu 14.04
+
+      To install basic system prerequisites, run the command:
+
+      .. code-block:: bash
+
+         sudo apt-get install -y python-pip python-virtualenv python-dev git
+
+      To install system prerequisites for Girder's ``ldap`` plugin, run the command:
+
+      .. code-block:: bash
+
+         sudo apt-get install -y libldap2-dev libsasl2-dev
+
+   .. group-tab:: RHEL (CentOS) 7
+
+      To install basic system prerequisites:
+
+        First, enable the `Extra Packages for Enterprise Linux <https://fedoraproject.org/wiki/EPEL>`_ YUM repository:
+
+        .. code-block:: bash
+
+           sudo yum -y install epel-release
+
+        Then, run the command:
+
+        .. code-block:: bash
+
+           sudo yum -y install python2-pip python-virtualenv gcc python-devel curl git
+
+      To install system prerequisites for Girder's ``ldap`` plugin, run the command:
+
+      .. code-block:: bash
+
+         sudo yum -y install openldap-devel cyrus-sasl-devel
+
+   .. group-tab:: macOS
+
+      Install `Homebrew <https://brew.sh/>`_.
+
+      To install all the prerequisites at once just use:
+
+      .. code-block:: bash
+
+         brew install python
```

Comment:
Curl, git, ldap?

Reply:
@zachmullen added this section. I don't personally have a local OSX environment to test what actually needs to be installed (and isn't provided by the base environment). If anyone else can actually confirm that other things needed to be added here, I'd suggest they add it in the future.